.github/workflows/cygwin.yml

```diff
@@ -216,7 +216,7 @@ jobs:
           cat mplsetup.cfg
           # All dependencies must have been pre-installed, so that the minver
           # constraints are held.
-          python -m pip install --no-deps -ve .
+          python -m pip install --no-deps -v --no-build-isolation --ediable .
```

Comment:
typo `--ediable` --> `--editable`

Reply:
Yes, also missed some of the configuration options.